addons/website/static/src/snippets/s_masonry_block/options.js

```diff
@@ -11,4 +11,21 @@ options.registry.MasonryLayout = options.registry.SelectTemplate.extend({
         this.containerSelector = '> .container, > .container-fluid, > .o_container_small';
         this.selectTemplateWidgetName = 'masonry_template_opt';
     },
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Change the container class according to the template.
+     *
+     * @see this.selectClass for parameters
+     */
+    containerClass: function (previewMode, widgetValue, params) {
+        const container = this.$target[0].firstElementChild;
+
+        // Remove classes, filter out falsy values, and safely add new class
+        container.classList.remove(...params.possibleValues.filter(Boolean));
+        container.classList.add(...[widgetValue].filter(Boolean));
```

Comment:
I've applied your patch, thanks 👍 


> why did you need to filter booleans ?

when using `possibleValues` and empty value is provided as well, triggering this error ⬇︎ 
It's not the case anymore since with your code we define `containerClasses` specifically.

```console
SyntaxError: Failed to execute 'remove' on 'DOMTokenList': The token provided must not be empty.
    Error: Failed to execute 'remove' on 'DOMTokenList': The token provided must not be empty.
```